packages/table/src/table.tsx

```diff
@@ -55,22 +54,22 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     children?: React.ReactElement<IColumnProps>;
 
     /**
-     * If true, empty space in the table container will be filled with empty
+     * If `true`, empty space in the table container will be filled with empty
      * cells instead of a blank background.
      * @default false
      */
     fillBodyWithGhostCells?: boolean;
 
     /**
-     * Used for hotkey copy, via (mod+c), as long as this property exists. 
-     * If exists, a callback that returns the data for a specific cell. This need not
-     * match the value displayed in the `<Cell>` component. The value will be
-     * invisibly added as `textContent` into the DOM before copying. 
+     * Used for hotkey copy, via `mod+c`, as long as this property exists. If it
+     * exists, a callback that returns the data for a specific cell. This need
```

Comment:
total rewrite proposal: 

> If defined, this callback will be invoked for each cell when the user attempts to copy
a selection via `mod+c`. The returned data will be copied to the clipboard and need not match
the display value of the `<Cell>`. The data will be invisibly added as `textContent` into
the DOM before copying. If not defined, keyboard copying via `mod+c` will be disabled.

cc @gscshoyru